torch/nn/modules/rnn.py

```diff
@@ -897,9 +896,9 @@ def forward(self, input, hx=None):  # noqa: F811
 
 
 class GRU(RNNBase):
-    r"""__init__(self,input_size,hidden_size,num_layers=1,bias=True,batch_first=False,dropout=0.0,bidirectional=False,device=None,dtype=None)
-
-    Applies a multi-layer gated recurrent unit (GRU) RNN to an input sequence.
+    r"""Apply a multi-layer gated recurrent unit (GRU) RNN to an input sequence.
```

Comment:
Same comment as above

Reply:
addressed in the latest commit - please confirm if it's okay.